### PR TITLE
all: fix sample events to agree with ECS

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

```
* cisco_umbrella
* forgerock
* lumos
* mattermost
* microsoft_exchange_online_message_trace
* pulse_connect_secure
* sentinel_one
* ti_cybersixgill
* trend_micro_vision_one

Also fix missed changelog entry link in crowdstrike package.
```

The first commit was generated with
```
for p in cisco_umbrella forgerock lumos mattermost microsoft_exchange_online_message_trace pulse_connect_secure sentinel_one ti_cybersixgill trend_micro_vision_one;
do (
  cd $p;
  elastic-package test system -g && \
  elastic-package build && \
  elastic-package changelog add \
    --type bugfix \
    --next patch \
    --description 'Fix sample event.' \
    --link https://github.com/elastic/integrations/pull/10136
); done
```

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- For #10135

### CODE REVIEW

File: packages/cisco_umbrella/data_stream/log/sample_event.json [hunk 1]
```diff
@@ -37,24 +37,26 @@
     },
     "data_stream": {
         "dataset": "cisco_umbrella.log",
-        "namespace": "ep",
+        "namespace": "27145",
```
Comment (kcreddy) [inline packages/cisco_umbrella/data_stream/log/sample_event.json:40]: Is this change related to new `elastic-package` version?

**Review by kcreddy — APPROVED** [review]
Should we be using `-beta` instead of bumping patch, as the change is related to running just the tests?

File: packages/cisco_umbrella/data_stream/log/sample_event.json (hunk 1 above)
Comment (efd6) [reply packages/cisco_umbrella/data_stream/log/sample_event.json:40]: Yes. I run ep at tip and this was changed in elastic/elastic-package@3cc9d9973937b3f864f914b4253f738f3b37d39b.
